layouts/partials/head-additions.html

```diff
@@ -5,9 +5,19 @@
 <link rel="prefetch" href="{{.NextInSection.Permalink }}">
 {{end}}
 
-{{/* TODO: This hides search until the integrated sites are live. It is safe to remove at that point. */}}
 <style media="screen">
+  /* TODO: This hides search until the integrated sites are live. It is safe to remove at that point. */
```

Comment:
I'm pretty sure the two approaches to commenting are not equal. I prefer the existing where the comments is not visible in the final source.